### PR TITLE
fix: fix resend email verification feature not sending email

### PR DESCRIPTION
**Pull request by djaiss**
This PR fixes the Resend verification email not working.

The route needs a POST and not a GET, therefore going to error.

We probably should modify the test as well.

### CODE REVIEW

File: resources/views/auth/login.blade.php [hunk 1]
```diff
@@ -57,15 +57,12 @@
...
-              @if (isset($errors))
-                @if (count($errors) > 0)
-                  <div class="form-group links">
-                    <ul>
-                      <li>{{ trans('auth.password_forget') }}&nbsp;<a href="{{ route('password.request') }}">{{ trans('auth.password_reset') }}</a></li>
-                    </ul>
-                  </div>
-                @endif
-              @endif
+              <div class="form-group links">
```
Comment (djaiss) [inline resources/views/auth/login.blade.php]: This is just a nice to have. I don't see why we should hide this link unless we make an error.

File: resources/views/auth/verify.blade.php [hunk 2]
```diff
@@ -22,7 +22,11 @@
               @endif
 
               {{ trans('auth.confirmation_check') }}<br/>
-              {!! trans('auth.confirmation_request_another', ['url' => route('verification.resend')]) !!}<br/>
+              {!! trans('auth.confirmation_request_another') !!}<br/>
+              <form action="{{ route('verification.resend') }}" method="post" class="di">
```
Comment (djaiss) [inline resources/views/auth/verify.blade.php]: This is the fix here. It needs to call a form that triggers a POST request, therefore the use of the <form> as well as the submit button.

File: resources/lang/en/auth.php [hunk 3]
```diff
@@ -73,7 +73,8 @@
     'confirmation_title' => 'Verify Your Email Address',
     'confirmation_fresh' => 'A fresh verification link has been sent to your email address.',
     'confirmation_check' => 'Before proceeding, please check your email for a verification link.',
-    'confirmation_request_another' => 'If you did not receive the email <a href=":url">click here to request another</a>.',
+    'confirmation_request_another' => 'If you did not receive the email',
```
Comment (asbiin) [inline resources/lang/en/auth.php]: I would rather write
```suggestion
    'confirmation_request_another' => 'If you did not receive the email <button type="submit" class=":class">click here to request another</button>.',
```

and use this single text in the form

File: resources/views/auth/login.blade.php (hunk 1 above)
Comment (asbiin) [reply resources/views/auth/login.blade.php]: OK why not

**Comment by asbiin** [comment]
@djaiss I've submitted an other solution.
The big green button on the middle of the page was really not nice.
This is the UI with this solution:
image

(yes there is no difference with the actual one)

**Comment by djaiss** [comment]
@asbiin awesome solution. let's merge this.

**Pull request merged**